server.js

```diff
@@ -1894,7 +1894,7 @@ if (config.startServer) {
         }
 
         if (config.logFilename) {
-          logger.addFileLogging(config.logFilename);
+          enableFileLogging(config.logFilename);
```

Comment:
Here I'd love to have:
```suggestion
          enableFileLogging(config.logFilename, 'debug');
          enableFileLogging(config.logErrorFilename, 'error');
```